FAQ.md

```diff
@@ -4,10 +4,28 @@ This sort of declaration is what other languages use (notably Java), and implies
 
 ## Why isn't access `this.x`?
 
-Having a private field named `x` must not prevent there from being a public field named `x` (see "encapsulation" below), so accessing a private field can't just be a normal lookup.
+Having a private field named `x` must not prevent there from being a public field named `x`, so accessing a private field can't just be a normal lookup.
 
 Less significantly, this would also break an invariant of current syntax: `this.x` and `this['x']` are currently always semantically identical.
 
+### Why can't a private field named `x` conflict with a public field named `x`?
```

Comment:
I liked the previous phrasing better, or maybe something like: "Why isn't a private field `#x` said to be in conflict with a public field `x`"? I think a typical mental model of our semantics would be that we *do* allow conflicts, and disambiguate in syntax by whether there's a `#`, so this question seems to be off by one negation.